### PR TITLE
LPS-179549 As an Instance Administrator, I want to configure Content-Security-Policy HTTP header to my instance

### PR DESCRIPTION
**Pull request by oliverkecskemety**
[LPS-179549](https://issues.liferay.com/browse/LPS-179549)

### CODE REVIEW

**Comment by oliverkecskemety** [comment]
ci:test:sf

**Comment by oliverkecskemety** [comment]
ci:test:relevant

**Comment by oliverkecskemety** [comment]
ci:test:sf

**Comment by oliverkecskemety** [comment]
ci:test:relevant

**Comment by oliverkecskemety** [comment]
ci:stop

**Comment by oliverkecskemety** [comment]
ci:test:sf

**Comment by oliverkecskemety** [comment]
ci:test:relevant

**Comment by JoyceWang08211** [comment]
ci:test:bundle

File: modules/apps/portal-security/portal-security-content-security-policy-test/src/test/java/com/liferay/portal/security/content/security/policy/internal/servlet/filter/ContentSecurityPolicyFilterTest.java [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+		Assert.assertFalse(
+			_mockHttpServletResponse.containsHeader(
+				ContentSecurityPolicyConstants.CONTENT_SECURITY_POLICY_HEADER));
+	}
+
+	@Test
+	public void testWhenConfiguredCorrectlyCspHeaderShouldBeAdded()
+		throws Exception {
+
+		String cspPolicy = "default-src 'self'";
```
Comment (gaborlovasdotcom) [inline modules/apps/portal-security/portal-security-content-security-policy-test/src/test/java/com/liferay/portal/security/content/security/policy/internal/servlet/filter/ContentSecurityPolicyFilterTest.java:110]: default-src 'self';

**Comment by JoyceWang08211** [comment]
ci:test:bundle

**Comment by stian-sigvartsen** [comment]
Continues in liferay-appsec/liferay-portal/pull/1118 .

**Pull request closed without merging**